app/controllers/application_controller.rb

```diff
@@ -7,4 +8,122 @@ class ApplicationController < Sinatra::Base
     set :views, 'app/views'
   end
 
+  enable :sessions
+
+  get '/tweets/new' do 
+    redirect :'/login' if !session[:id]
+    erb "<form action='/tweets/new' method='post'>
+    <input type='text' name='content'>Tweet</input>
+    <input type='submit' id='submit'>
+    </form>"
+  end
+
+  post '/tweets/new' do 
```

Comment:
Typically, tweet creation would be posted to `/tweets` instead of `/tweets/new`.